io.github.sil_car.SqueezeVid.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Version=1.0
+Name=Squeeze Vid
+NoDisplay=true
+Comment=CLI app
+Icon=io.github.sil_car.SqueezeVid
+Exec=/app/bin/squeeze-vid
```

Comment:
```suggestion
Exec=squeeze-vid
```

Reply:
There's no risk of that executing a different command with the same name from PATH, right, since this is within the flatpak's own confinement? I just want to make sure, because I have the same command already in PATH.